### PR TITLE
SecureDrop 1.7.0-rc4

### PR DESCRIPTION
**Pull request by sssoleileraaa**
## Status

Ready for review

## Description of Changes

Contains the updated changelog (will consolidate in the final release) and version strings, towards #5689

### CODE REVIEW

**Review by zenmonkeykstop — APPROVED** [review]
Revised changelog lgtm, good to go when CI passes

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 1.7.0~rc4
+
+* Update v2 deprecation warning date from February to March
```
Comment (conorsch) [inline changelog.md:5]: For reference, https://github.com/freedomofpress/securedrop/pull/5734. We'll add that in the final changelog

**Review by conorsch — APPROVED** [review]

File: changelog.md (hunk 1 above)
Comment (sssoleileraaa) [reply changelog.md:5]: good catch

Comment (sssoleileraaa) [reply changelog.md:5]: for the final release, i'll have to move these bullets (or merge them with existing bullets) to the 1.7.0 changelog section so we'll come across this again

**Pull request merged**